cffi.asd

```diff
@@ -65,11 +65,28 @@
      (:file "foreign-vars")
      (:file "features")))))
 
-(defmethod operation-done-p ((o test-op) (c (eql (find-system :cffi))))
-  nil)
-
-(defmethod perform ((o test-op) (c (eql (find-system :cffi))))
-  (operate 'asdf:load-op :cffi-tests)
-  (operate 'asdf:test-op :cffi-tests))
+(defsystem :cffi/c2ffi
```

Comment:
Sounds like a good idea. We can probably figure out a way to put some sort of deprecation warning when loading those empty systems.